### PR TITLE
Add wallet name feat

### PR DESCRIPTION
**Pull request by Tommytrg**
close #1482

### CODE REVIEW

File: src/store/wallet.js [hunk 1]
```diff
@@ -782,12 +790,13 @@ export default {
 
     createWallet: async function(context, params) {
       const request = await context.state.api.createWallet({
-        name: 'first',
-        caption: '1',
+        title: context.state.title,
```
Comment (gabaldon) [inline src/store/wallet.js:793]: The correct param is 'name' not 'title'.

File: src/components/steps/WalletDescription.vue [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    },
+    nextRoute() {
+      return `/ftu/encryption-pass${this.isImporting ? '?import=true' : ''}`
+    },
+    previousRoute() {
+      return this.isImporting ? '/ftu/import-wallet' : '/ftu/seed-validation'
+    },
+    title: {
+      set(val) {
+        this.$store.commit('setWalletDescription', { title: val })
```
Comment (gabaldon) [inline src/components/steps/WalletDescription.vue:57]: Could you move this to a computed using ...mapState({}) ?

**Review by gabaldon — COMMENTED** [review]
You should also add the wallet name to the `walletList` component and to the `NetworkStatus`.

**Review by gabaldon — CHANGES_REQUESTED** [review]
The wallet name should also appear in the `NetworkStatus` component.

File: src/store/wallet.js [hunk 3]
```diff
@@ -873,6 +874,7 @@ export default {
       const request = await context.state.api.getWalletInfos()
 
       if (request.result) {
+        console.log('result:>', request.result)
```
Comment (tmpolaczyk) [inline src/store/wallet.js:877]: :)

```suggestion
```

File: tests/unit/src/components/steps/WalletDescription.spec.js [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+      })
+
+      expect(
+        wrapper
+          .findAllComponents(Input)
+          .at(0)
+          .props().value,
+      ).toBe('random text')
+    })
+    // console.log('AAA', a.props().type)
```
Comment (tmpolaczyk) [inline tests/unit/src/components/steps/WalletDescription.spec.js:83]:
```suggestion
```

File: src/components/NetworkStatus.vue [hunk 5]
```diff
@@ -122,7 +122,11 @@ export default {
     }
   },
   computed: {
-    ...mapGetters(['network']),
+    ...mapGetters(['network', 'unlockedWallet']),
+    walletName () {
+      console.log('--->', this.unlockedWallet)
```
Comment (tmpolaczyk) [inline src/components/NetworkStatus.vue:127]:
```suggestion
```

File: src/api.js [hunk 6]
```diff
@@ -359,3 +359,16 @@ function computeTransactionAddress(inputs, outputs, type) {
     return outputs.length > 2 ? 'several addresses' : outputs[0].address
   }
 }
+
+function standardizeWalletInfos (response) {
+  console.log('response', response)
```
Comment (gabaldon) [inline src/api.js:364]: Please remove console.log

File: src/components/steps/WalletDescription.vue [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+    ...mapMutations(['setWalletDescription']),
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/theme.scss';
+
+.paragraph {
+  margin-bottom: 32px;
```
Comment (gabaldon) [inline src/components/steps/WalletDescription.vue:102]:
```suggestion
  margin-bottom: 16px;
```

**Review by gabaldon — CHANGES_REQUESTED** [review]

**Review by gabaldon — APPROVED** [review]

**Pull request merged**